src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1
+  .then((message) => {
+    const successMessage = document.createElement('div');
+
+    successMessage.classList.add('message');
+    successMessage.textContent = message;
+    document.body.append(successMessage);
+  })
+  .catch(() => {
+    const errorMessage = document.createElement('div');
+
+    errorMessage.classList.add('message');
+    errorMessage.classList.add('error-message');
+    errorMessage.textContent = 'Promise was rejected!';
+    document.body.append(errorMessage);
+  });
```

Comment:
The catch block for `promise1` will never execute because the promise only resolves on a click event and does not have a reject path. Consider removing this catch block or adding a reject path to `promise1` if needed.